### ISSUE TITLE
Using <script global> or `:global` selector doesn't silence eslint warnings

### ISSUE THREAD

**Issue by dtzxporter** [sveltejs/eslint-plugin-svelte3#123]
Eslint will still complain about an 'unused' selector when svelte's linter knows that it's marked global.

```html
<script global>
body { color: red; }
</script>
```

**Comment by jcayzac** [comment]
@dtzxporter what is your eslint confguration?

**Comment by Conduitry** [comment]
I assume this is meant to say `<style global>`? This isn't an official Svelte syntax, and is something added by svelte-preprocess. But, since that's an official library, I think it might make sense to automatically suppress warnings about the CSS in that case. `:global()` should already be working, because we're relying on the Svelte compiler itself for these warnings.
